package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "opensearch-security-dashboards",
-  "version": "1.3.0.0",
+  "version": "1.3.11.0",
```

Comment:
Should the version on 1.x be referring to 1.4?

Reply:
Unclear, but I'd say out of scope to get the 1.x branch unblocked.